templates/articles/article_page.html

```diff
@@ -10,7 +10,7 @@
 {% block breadcrumb %}{% endblock %}
 {% block content %}
     {% include 'includes/generic-intro.html' with breadcrumb=True title=page.title intro=page.intro %}
-    {% include 'includes/hero-img.html' %}
+    {% include 'includes/hero-img.html' with hero_image=page.hero_image caption=page.hero_image_caption %}
```

Comment:
Personally, I would rather this just be `{% include "includes/page-hero-img.html" %}` and the template grab the necessary values from `page`. Is there a reason for this that I'm not getting?

Reply:
Happy with the explanation. Let's keep this as is